src/classic.js

```diff
@@ -36,8 +32,7 @@ export default class ClassicEditor extends StandardEditor {
 
 		this.data.processor = new HtmlDataProcessor();
 
-		this.ui = new ClassicEditorUI( this );
-		this.ui.view = new BoxedEditorUIView( this.locale );
+		this.ui = new ClassicEditorUIView( this, this.locale );
```

Comment:
But it is the view... It's just "editing's view", not "UI's view". There can be multiple views for multiple things in your application. View != UI.